se.cendio.tlclient.metainfo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
you also need screenshots.

Reply:
The problem is that upstream doesn't really provide any screenshots. The closest would be this: https://www.cendio.com/website/var/assets/firstpage/anatomy_thinlinc_session.png

Reply:
We've produced a set of screenshots now:

https://www.cendio.com/resources/images/screenshots/thinlinc/linux-en-intro.png
https://www.cendio.com/resources/images/screenshots/thinlinc/linux-en-local-drives.png
https://www.cendio.com/resources/images/screenshots/thinlinc/linux-en-fullscreen-mode.png
https://www.cendio.com/resources/images/screenshots/thinlinc/linux-en-profile-chooser.png
https://www.cendio.com/resources/images/screenshots/thinlinc/linux-en-login-window.png
https://www.cendio.com/resources/images/screenshots/thinlinc/linux-en-graphics-acceleration.png 

Our intent is to keep those URLs stable for things like flatpaks. The above order is also the order they're intended to be shown.

Reply:
Thanks! I've integrated them and used the captions on the images as captions in the manifest if that works for you. The screenshots look great!

Reply:
Also, there's a test build available now down below.